src/raylib.h

```diff
@@ -1104,6 +1104,7 @@ RLAPI bool IsMouseButtonUp(int button);                       // Check if a mous
 RLAPI int GetMouseX(void);                                    // Get mouse position X
 RLAPI int GetMouseY(void);                                    // Get mouse position Y
 RLAPI Vector2 GetMousePosition(void);                         // Get mouse position XY
+RLAPI Vector2 GetMouseDelta();                                // Mouse movement per frame XY
```

Comment:
Please, could you review the function description to be consistent with all the other related function descriptions? -> `Get mouse delta between frames`.